### PR TITLE
add unitest for colo_tensor 1DTP cross_entropy

### PR DESCRIPTION
**Pull request by 1SAA**

### CODE REVIEW

**Review by feifeibear — APPROVED** [review]

File: colossalai/nn/loss/loss_1d.py [hunk 1]
```diff
@@ -10,19 +11,19 @@ class _VocabParallelCrossEntropy1D(torch.autograd.Function):
 
     @staticmethod
     @custom_fwd(cast_inputs=torch.float32)
-    def forward(ctx, vocab_parallel_logits, targets):
+    def forward(ctx, vocab_parallel_logits, targets, process_group):
+        if process_group is None:
+            process_group = gpc.get_group(ParallelMode.PARALLEL_1D)
```
Comment (feifeibear) [inline colossalai/nn/loss/loss_1d.py:16]: 现在不要怕用gpc的 process_group了！

**Comment by 1SAA** [comment]
image
